config-model/src/main/java/com/yahoo/schema/processing/SummaryElementsSelectorValidator.java

```diff
@@ -74,9 +73,7 @@ private void verifySelectElementsBy(DocumentSummary summary, SummaryField field,
         if (!found) {
```

Comment:
Problem is on the previous line; the "s" has just "elementwise" as its name returned by .getName(); use .toString() instead to also get the arguments.

Reply:
Of course. Thanks!